### PR TITLE
set my api and created add blog, get a single blog, get all blogs end…

### PR DESCRIPTION
**Pull request by jimmymut**
… points

made all routes endpoints, refactored my codes to routes-controller-modle, validated some endpoints with joi and refactored to es6

added authentication on some routes

### CODE REVIEW

**Comment by jimmymut** [comment]
I am going to work on the comments you just said on

File: src/authentications/auth.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import passport from "passport";
+import Admin from "../models/adminModel";
+import { Strategy as JwtStrategy } from "passport-jwt";
+import { ExtractJwt as ExtractJwt } from "passport-jwt";
+
+const opts = {};
+opts.jwtFromRequest = ExtractJwt.fromAuthHeaderAsBearerToken();
+opts.secretOrKey = process.env.SECRET_KEY;
+// opts.issuer = 'accounts.examplesoft.com';
```
Comment (Eli250) [inline src/authentications/auth.js:9]: Remove these comments

File: src/middlewares/signInAuth.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// import passport from "passport";
```
Comment (Eli250) [inline src/middlewares/signInAuth.js:1]: Delete all unused files.

**Review by Eli250 — CHANGES_REQUESTED** [review]
Work on all comments.

**Pull request merged**